_posts/en/newsletters/2021-05-05-newsletter.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Bitcoin Optech Newsletter #147'
+permalink: /en/newsletters/2021/05/05/
+name: 2021-05-05-newsletter
+slug: 2021-05-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter encourages miners to start signaling for taproot
+and describes continued discussion about closing lost LN channels using
+only a wallet seed.  Also included are our regular sections with
+announcements of releases and release candidates, plus notable changes
+to popular Bitcoin infrastructure software.
+
+## Action items
+
+- **Miners encouraged to start signaling for taproot:** miners who
+  expect to be willing to enforce the new consensus rules for
+  [taproot][topic taproot] are encouraged to begin signaling and to
+  ensure they'll be able to run Bitcoin Core 0.21.1 (described below) or
+  other compatible taproot-enforcing software by the minimum activation
+  block specified in [BIP341][].
+
+    Anyone wanting to trustlessly monitor signaling progress can upgrade
+    to Bitcoin Core 0.21.1 and use the `getblockchaininfo` RPC.  For
+    example, the following command line prints the number of blocks in
+    the current retarget period, the number of those blocks which have
+    signaled, and whether it's possible for taproot to activate in this
+    period (assuming there's no reorg):
+
+    ```text
+    $ bitcoin-cli getblockchaininfo \
+      | jq '.softforks.taproot.bip9.statistics | .elapsed,.count,.possible'
+    353
+    57
+    false
+    ```
+
+    If you prefer a graphical representation with supplementary
+    information about miner progress and don't need to use your own
+    node, we recommend [Taproot.watch][] by Hampus Sjöberg.
+
+## News
+
+- **Closing lost channels with only a BIP32 seed:** as described in [Newsletter
+  #128][news128 ln ecdh], Lloyd Fournier proposed a method for creating
+  new channels that would allow a user who lost all information except
+  for their BIP32 wallet seed to re-discover their peers using only
+  public information about the LN network.  Once the user found their
+  peers, they could request the peers close their mutual channels using
+  the [BOLT2][] data loss protection protocol (see [Newsletter
+  #31][news31 data_loss]).  The proposed method isn't perfect---users
+  should still create backups[^missing-peer] and replicate them across
+  independent systems---but Fournier's proposal provides additional
+  redundancy that would be especially useful for everyday users.
+
+    A couple weeks ago, Rusty Russell restarted the [thread][russell
```

Comment:
I know that "a couple weeks" can be used in American English, but I don't know whether it's considered informal or not (in British English we'd always use "a couple of weeks"). I see a mixture of "a couple {things}" and "a couple of {things}" in previous newsletters.